fe/fe-core/src/test/java/org/apache/doris/qe/StmtExecutorTest.java

```diff
@@ -801,4 +802,59 @@ public void testUseWithCatalogFail(@Mocked UseStmt useStmt, @Mocked SqlParser pa
 
         Assert.assertEquals(QueryState.MysqlStateType.ERR, state.getStateType());
     }
+
+    @Test
+    public void testBlockSqlAst(@Mocked UseStmt useStmt, @Mocked SqlParser parser) throws Exception {
```

Comment:
test `create file`, `create function` statment too

also test some commands that should not be blocked